keras_cv/models/convmixer.py

```diff
@@ -106,20 +107,24 @@ def apply_conv_mixer_layer(x, dim, kernel_size):
     Returns:
         the updated input tensor.
     """
+    if name is None:
+        name = str(backend.get_uid("apply_conv_mixer_layer"))
```

Comment:
This breaks our pretrained weights, and `apply_` should not end up in layer names.

Reply:
removed apply_ from names

Reply:
> This breaks our pretrained weights, and `apply_` should not end up in layer names.

@LukeWood , then name string should not be generated this way if name is `None` ?